src/api/routes/notificationRouter.js

```diff
@@ -0,0 +1,33 @@
+import { Router } from 'express';
+import notificationController from '../controllers/notificationController';
+import checkValidToken from '../../middlewares/checkValidToken';
+import validateInputs from '../../middlewares/validations/body.inputs';
+import bodyVerifier from '../../middlewares/validations/body.verifier';
+import checkUpdateNotification from '../../middlewares/check.update.notification';
+
+const router = new Router();
+
+router.patch('/configuration',
+  checkValidToken,
+  bodyVerifier,
+  validateInputs('notificationConfig', ['inApp']),
+  notificationController.updateConfig);
+
+router.get('/configuration', checkValidToken, notificationController.getConfig);
+
+router.get('/', checkValidToken, notificationController.getAll);
+router.get('/seen', checkValidToken, notificationController.getAll);
+router.get('/unseen', checkValidToken, notificationController.getAll);
+
+router.patch('/:id/seen', checkValidToken, checkUpdateNotification, notificationController.update);
+
+router.patch('/:id/unseen',
+  checkValidToken,
+  checkUpdateNotification,
+  notificationController.update);
+
+router.get('/configuration/unsubscribe/email',
```

Comment:
is it get?

Reply:
Yes @alainburindi 
Because it will only be coming from the `email` sent to the user. 
If you have another suggestion, I'll be glad to implement it